plugins/module_utils/redfish_utils.py

```diff
@@ -2187,8 +2191,12 @@ def _find_empty_virt_media_slot(resources, media_types,
             else:
                 if media_match_strict:
                     continue
-            # if ejected, 'Inserted' should be False
-            if (not data.get('Inserted', False)):
+            # Base on current XCC capability, slot RDOC1/2 and Remote1/2/3/4 are not supported to Insert/Eject.
+            if ("/Systems/1/" in uri or "/Manager/1/" in uri) and ('RDOC' in uri or 'Remote' in uri):
```

Comment:
Is this correct for all kind of Redfish devices out there?

Reply:
We may use another way to identify lenovo XCC, like 'Vendor' in /redfish/v1.

Reply:
I would also avoid hard coding things like "RDOC" and "Remote" as methods to detect the capability of setting the image. We may need to think of different logic to determine when a slot cannot be modified. I was hoping the `Allow` HTTP header would be able to tell us this, but it seems like you can PATCH other properties on this type of virtual media resource. I'll ask around if we can think of more hardened logic for this.